src/wordy_webserver_response.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+class WebServerResponse {
+  constructor(storage) {
+    this.storage = storage;
+  }
+
+  static getHome() {
+    return 'WordyBot';
+  }
+
+  processCommand(req, callback) {
+    const command = req.body.command;
+    const userId = req.body.user_id;
+
+    console.log('Command received');
+    console.log(`Command: ${command}`);
+    console.log(`User ID: ${userId}`);
+
+    switch (command) {
+      case '/wordy-in':
+        this.userIn(userId, callback);
+        break;
+      case '/wordy-out':
+        this.userOut(userId, callback);
+        break;
+      default:
+        console.warn('Unknown command, ignoring');
+        callback('Unknown command.');
+        break;
+    }
+  }
+
+  userIn(userId, callback) {
```

Comment:
No worries, leave it to me for now.

I still have to put in place the auth implementation via middleware. It will likely give me pointers to more natural JS / Node / Express architectures.